docs/content/users/extend/additional-services.md

```diff
@@ -87,6 +87,7 @@ The `install.yaml` is a simple YAML file with a few main sections:
 In any stanza of `pre_install_actions` and `post_install_actions` you can:
 
 * Use `#ddev-description:<some description of what stanza is doing>` to instruct DDEV to output a description of the action it's taking.
+* Use `#ddev-warning-exit-code:<an integer between 1-255>` to instruct DDEV to consider a certain exit code of the stanza as a warning, not an error, continuing with other actions as normal.
```

Comment:
Is it possible to accept multiple exit codes?

Reply:
I have actually thought about this but I figured it made no sense, if your use case requires different exit codes you can bundle them together into a single one in your action:

```bash
...
ret1=$?
...
ret2=$?
if [[ "$ret1" == "2" || "$ret2" == "3" ]]; then
  exit 127
fi
```